Command/BuildCommand.php

```diff
@@ -82,6 +82,7 @@ public function configure()
 
         $this->addArgument('target', InputArgument::OPTIONAL, 'Target to build', null);
         $this->addOption('nodeps', 'D', InputOption::VALUE_NONE, 'Ignore dependencies');
+        $this->addOption('exitCode', 'e', InputOption::VALUE_NONE, 'Keep the exit code of a job if it fails');
```

Comment:
The `e` is reserved for `--env` in Symfony so we can not add a short `e` for this argument.
Also the name should be with `-` like symfony uses `no-interaction. 

I would would in this case go with `--keep-exit-code` and so `-k`. But wait what the others say to this.

/cc @nnatter

Reply:
sounds good to me 🙂